sql.mdx

```diff
@@ -309,6 +309,55 @@ param_regex =\$\{[^}]*\}
 1 = 'arbitrary_placeholder_name'
 ```
 
+### Generated SQL file resolvers
+You can programmatically generate SQL resolvers with [`make_sql_file_resolver`](/api-docs#make_sql_file_resolver). This
```

Comment:
the api docs link actually does not exist yet; I believe it will exist in the next release along with this, since I added it to the TOC